client/v3/naming/endpoints/endpoints_impl.go

```diff
@@ -116,6 +143,19 @@ func (m *endpointManager) NewWatchChannel(ctx context.Context) (WatchChannel, er
 }
 
 func (m *endpointManager) List(ctx context.Context) (Key2EndpointMap, error) {
```

Comment:
Thank you for implementing this. 

So if we have a complementary methods (Add, Delete, Update, List), 
could you, please, add integration test for endpoints alone (just that DB operations behave as expected).

Reply:
Done